src/action/sample.js

```diff
@@ -0,0 +1,14 @@
+// Moduler Importations
+import { SAMPLE_LOGIN_SUCCESS } from "../actionTypes/sample";
+
+/**
+ * @param {object} score 
+ * @desc sample of an action
+*/
+export const setPlayerScore = score => {
+    return {
+      type: SAMPLE_LOGIN_SUCCESS,
+      score: score
+    };
+  };
+ 
```

Comment:
#### Critical
New line here???

Reply:
There should be a new line at the end of the file.

Reply:
<img width="741" alt="screen shot 2018-10-31 at 1 35 41 pm" src="https://user-images.githubusercontent.com/20081126/47788428-fe928d80-dd11-11e8-8e85-a187088060dc.png">

Reply:
Line 14 is the new line at the end of the file

Reply:
Have you pushed this cos github still flags it?
<img width="405" alt="screen shot 2018-10-31 at 1 38 18 pm" src="https://user-images.githubusercontent.com/26272628/47788550-54ffcc00-dd12-11e8-9d6a-e01d36a59ed0.png">

Reply:
Yes the screenshot is from the PR online